jerry-core/ecma/builtin-objects/ecma-builtin-date.c

```diff
@@ -227,32 +270,29 @@ ecma_date_construct_helper (const ecma_value_t *args, /**< arguments passed to t
 
     if (ECMA_IS_VALUE_ERROR (status))
     {
-      return status;
+      return false;
     }
   }
 
-  ecma_number_t prim_value = ecma_number_make_nan ();
-
   if (!ecma_number_is_nan (date_nums[0]))
   {
     /* 8. */
-    ecma_number_t y = ecma_number_trunc (date_nums[0]);
+    ecma_number_t yi = ecma_number_trunc (date_nums[0]);
```

Comment:
yi is coming from new spec, isn't it? In this case please update references too to the latest spec.

Reply:
Thanks, updated.